src/dfeed/web/web/view/index.d

```diff
@@ -134,7 +134,7 @@ void discussionIndexHeader()
 				) ~ `</li>`;
 		else
 			bits[0] ~= `<li>` ~
-				_!`You can read and post on this forum without %screating an account%s, but doing so offers %sa few benefits%s.`
+				_!`You can read and post on this forum without %screating an account%s, but creating an account offers %sa few benefits%s.`
```

Comment:
I trust you that repetition is an improvement here :)

Reply:
The problem with the old phrase was "doing so" might mean "without creating an account".

Reply:
Reading this again, I am now seeing that "offers a few benefits" here means "doesn't offer much", which is proof in itself that the sentence can be made better. For example, if we decide to keep the repetition, the sentence could end with "... offers some benefits."